src/main/java/tc/oc/pgm/listeners/MotdListener.java

```diff
@@ -31,7 +31,9 @@ public MotdListener() {
     this.format = Config.Motd.format();
 
     // Ensure there are always default keys
-    final MapInfo map = PGM.get().getMapOrder().getNextMap();
+    MapInfo map = PGM.get().getMapOrder().getNextMap();
+    if (map == null) map = PGM.get().getMapLibrary().getMaps().next();
```

Comment:
Is this assuming a map if a next map is missing? 
Next map not being defined is completely normal (for example, on voted pools) and shouldn't be avoided or treated like an error.

Reply:
If we don't know what the true next map will be, what should the motd say? `> PGM <` ?

Reply:
Either the current map, or something else like `Deciding next map`, `PGM`, or whatever other placeholder